public/components/experiment_listing/experiment_listing.tsx

```diff
@@ -41,6 +41,8 @@ export const ExperimentListing: React.FC<ExperimentListingProps> = ({ http, hist
   const [experimentToDelete, setExperimentToDelete] = useState<any>(null);
   const [refreshKey, setRefreshKey] = useState(0);
 
+  const DISABLED_BACKEND_PLUGIN_MESSAGE = 'Search Relevance Workbench is disabled';
```

Comment:
feels like, if we are going to have a constant for a text message, that this should be elsewhere?   Since we define this over and over?

Reply:
moved it to `common/index.ts`

no idea if that's the best place for it.